### PR TITLE
milestone 3 done

### PR DESCRIPTION
**Pull request by azdrenymeri**

### CODE REVIEW

File: src/components/App.css [hunk 1]
```diff
@@ -1,19 +1,47 @@
 .App {
```
[the remark this answers is not in the document]
Comment (azdrenymeri) [reply src/components/App.css]: Fixed

File: src/components/App.css [hunk 2]
```diff
@@ -1,19 +1,47 @@
...
-  width: 350px;
+  display: flex;
+  height: 50px;
+  width: 20%;
   font-size: 20px;
   padding: 0 5px 0 5px;
+  text-align: right;
+}
+.ButtonPanel {
+  display: flex;
```
[the remark this answers is not in the document]
Comment (azdrenymeri) [reply src/components/App.css]: changed to 100px

File: src/components/ButtonPanel.js [hunk 3]
```diff
@@ -5,32 +5,32 @@ class ButtonPanel extends Component {
     render(){
         return (
             <div className="ButtonPanel">
-                <div>
-                    <div>
+                <div className="ButtonPanel-Groups">
+                    <div className="ButtonPanel-Group">
                         <Buton name="AC" />
                         <Buton name="+/-" />
```
[the remark this answers is not in the document]
Comment (azdrenymeri) [reply src/components/ButtonPanel.js]: Added two more properties.
Also added as required props

File: src/components/ButtonPanel.js [hunk 4]
```diff
@@ -5,32 +5,32 @@ class ButtonPanel extends Component {
...
                     </div>
-                    <div>
+                    <div className="ButtonPanel-Group">
                         <Buton name="1" />
                         <Buton name="2" />
                         <Buton name="3" />
                         <Buton name="+" />
                     </div>
-                    <div>
+                    <div className="ButtonPanel-Group">
```
[the remark this answers is not in the document]
Comment (azdrenymeri) [reply src/components/ButtonPanel.js]: Done

File: src/components/Button.js [hunk 5]
```diff
@@ -8,16 +8,23 @@ class Button extends Component {
...
         )
     }
 }
 
 Button.propTypes = {
-    name: PropTypes.string.isRequired
+    name: PropTypes.string.isRequired,
+    color: PropTypes.string.isRequired,
+    wide: PropTypes.bool.isRequired
 }
```
Comment (SunnyAsar) [inline src/components/Button.js]: Hi, using default props to set the colour for  orange buttons is a requirement.
* [ ] defualt color for button is orange.

for the buttons that are orange, don't send a color prop but let the default color kick in.

Comment (azdrenymeri) [reply src/components/Button.js]: Added the defaul prop

**Pull request merged**